block/manager.go

```diff
@@ -936,6 +925,28 @@ func (m *Manager) publishBlock(ctx context.Context) error {
 	return nil
 }
 
+func (m *Manager) sign(payload []byte) ([]byte, error) {
```

Comment:
Per @tzdybal comment, we should have a unit test for this.

Reply:
done.